app/core/search.py

```diff
@@ -219,7 +219,9 @@ def process_result_into_csv(
                     "Document Title": document_title,
                     "Document URL": f"{url_base}/{document.slugs[-1].name}",
                     "Document Content URL": document_content,
-                    "Document Type": document.document_type,
+                    "Document Type": cast(
+                        str, document.valid_metadata["type"][0]  # type:ignore
```

Comment:
ditto - can we check we have a type - otherwise I think the search will 500 if we have bad data.